pyproject.toml

```diff
@@ -0,0 +1,388 @@
+[build-system]
+requires = ["setuptools>=61.0", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "otf-addons-gcp"
+version = "v0.1.0"
+authors = [{ name = "Aldo Troiano", email = "a.troiano@reply.com" }]
+license = { text = "GPLv3" }
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
+    "Operating System :: POSIX",
+]
+keywords = ["automation", "task", "framework", "gcp", "cloudstorage", "otf"]
+dependencies = [
+    "google-cloud-storage >= 2.17.0",
```

Comment:
I don't think this is used. Can it be removed?

Reply:
removed cloud-storage lib - keeping google-auth